tests/features/step_definitions_test.go

```
@@ -284,11 +284,11 @@ func (a *apiFeature) startLocalServer(port int) error {
 	if err != nil {
 		return err
 	}
-	validate := validation.NewValidator()
-	version, err := testhelpers.RepoVersion()
+	validate, err := validation.NewValidator()
 	if err != nil {
-		return logError(fmt.Errorf("read VERSION: %w", err))
+		return logError(err)
 	}
+	version, err := testhelpers.RepoVersion()
```

golangci-lint (2.12.2)

[error] ineffectual assignment to err

(ineffassign)